kiwi/iso_tools/iso.py

```diff
@@ -154,15 +154,15 @@ def relocate_boot_catalog(self, isofile):
                     new_volume_id = Iso._sub_string(
                         data=new_volume_descriptor, length=7
                     )
-                    if (bytes(b'TEA01') in new_volume_id or
-                            sector + 1 == ref_sector):
+                    if (
+                        bytes(
+                            b'TEA01'
+                        ) in new_volume_id or sector + 1 == ref_sector
+                    ):
```

Comment:
I am not sure about the original complain, but I guess that could be fixed moving a bit on the left the original line 158